test/test-cases.ts

```diff
@@ -0,0 +1,95 @@
+import { Metric } from '../src/types';
```

Comment:
I think this can also go. A lot of these no longer apply upstream too, but that's another discussion.